addons/sale/models/account_move.py

```diff
@@ -63,14 +63,14 @@ def action_post(self):
         res = super(AccountMove, self).action_post()
         down_payment_lines = self.line_ids.filtered('is_downpayment')
         for line in down_payment_lines:
+            if not line.sale_line_ids.display_type:
```

Comment:
```suggestion
            if not any(sol.display_type for sol in line.sale_line_ids):
```
to be sure it doesn't break in some special cases where the aml could have multiple solines as it kinda seems supported considering the line below.

Reply:
Hello @Feyensv! Thank you for your review! I committed the changes you suggested.